### DISCUSSION TITLE
DWSIM 8 and Python Env

### DISCUSSION THREAD

**Discussion by kookma** [DanWBR/dwsim#334] [General]
With DWSIM 8 a rather large file is distributed (237MB) which is called PythonEnvs_win64.exe

I could not find any info on installation page except here: https://dwsim.org/index.php/download/ which states
Extract the Python Environments add-on to DWSIM’s installation directory.

What is this big file?
Is it required by normal users to be installed?
Should it be installed on any new update?

**Comment by DanWBR** [comment]
@kookma this file is a 7z-compressed file that contains two portable python environments, one of them containing thermo/chemicals and opem libraries and the other is for reaktoro. It is a separate file because you won't need to download it on every update, just when I decide to update the python environments themselves.

**Reply by kookma** [reply]
Thank you! It would be great to have this description on download page.

**Reply by DanWBR** [reply]
I'm still figuring it out, as you may have noticed that v8 wasn't released for Linux and macOS yet. Python works differently on such systems.
